gatilegrid/tilegrids.py

```diff
@@ -231,6 +231,11 @@ def getResolution(self, zoom):
         "Return the image resolution at a given zoom level"
         return self.tileSize(zoom) / self.tileSizePx
 
+    def getZoom(self, resolution):
+        "Return the zoom level for a given resolution"
+        assert resolution in self.RESOLUTIONS
+        return self.RESOLUTIONS.index(resolution)
```

Comment:
This is a bit weak for web mercator and tms as we have a couple of decimals.

getClosestZoom could be nice.